features/car/side_bias.feature

```diff
@@ -76,3 +74,45 @@ Feature: Testbot - side bias
            | a,d       | ab,cd,cd | depart,roundabout turn left exit-1,arrive     |
            | a,f       | ab,ef,ef | depart,roundabout turn straight exit-2,arrive |
            | a,h       | ab,gh,gh | depart,roundabout turn right exit-3,arrive    |
+
+
+    Scenario: Left-hand bias via location-dependent tags
+        Given the profile "car"
+        And the node map
+            """
+            a   b   c
+
+                d
+            """
+        And the ways with locations
+            | nodes |
+            | ab    |
+            | bc    |
+            | bd    |
+        And the extract extra arguments "--location-dependent-data test/data/regions/null-island.geojson"
+
+        When I route I should get
+            | from | to | route    | time       |
+            | d    | a  | bd,ab,ab | 24s +-1    |
+            | d    | c  | bd,bc,bc | 27s +-1    |
+
+
+    Scenario: Left-hand bias via OSM tags
+        Given the profile "car"
+        And the node map
+            """
+            a   b   c
+
+                d
+            """
+        And the ways with locations
+            | nodes | driving_side |
+            | ab    | right        |
```

Comment:
there some [residential roads](https://www.openstreetmap.org/way/134124277), [service roads](https://www.openstreetmap.org/way/34420205), [private ways]( https://www.openstreetmap.org/way/384394384)  or [u-turns](https://www.openstreetmap.org/way/364692295). Yes, we can add in future such handling, atm tags on ways have higher priority than default values.